Content.Client/Overlays/ShowHealthIconsSystem.cs

```diff
@@ -14,11 +16,18 @@ namespace Content.Client.Overlays;
 public sealed class ShowHealthIconsSystem : EquipmentHudSystem<ShowHealthIconsComponent>
 {
     [Dependency] private readonly IPrototypeManager _prototypeMan = default!;
+    [Dependency] private readonly MobStateSystem _mobState = default!;
 
     public HashSet<string> DamageContainers = new();
 
     [ValidatePrototypeId<StatusIconPrototype>]
-    private const string HealthIconFine = "HealthIconFine";
+    private const string HealthIconHealthy = "HealthIconHealthy";
+
+    [ValidatePrototypeId<StatusIconPrototype>]
+    private const string HealthIconCrit = "HealthIconCrit";
+
+    [ValidatePrototypeId<StatusIconPrototype>]
+    private const string HealthIconDead = "HealthIconDead";
 
     public override void Initialize()
```

Comment:
Instead of adding a bunch of constants just tie it to the MobState enum directly e.g. via dictionary, then you can just check the mobstate below instead of adding a bunch of if branches.